frontend/src/Buildings/buildingReducer.js

```diff
@@ -13,11 +16,15 @@ const initialState = {
 export default function buildings(state = initialState, action) {
   switch (action.type) {
     case FETCH_BUILDINGS_REQUEST:
+    case ADD_BUILDING_REQUEST:
       return { ...state, isLoading: true };
     case FETCH_BUILDINGS_SUCCESS:
       return { ...state, listOfBuildings: action.payload, isLoading: false };
     case FETCH_BUILDINGS_FAILURE:
+    case ADD_BUILDING_FAILURE:
```

Comment:
this would never happen because right now you don't have an action for this in the actionCreator